batchflow/batch_image.py

```diff
@@ -355,8 +355,8 @@ def _to_array_(self, image, dtype=None, channels='last'):
         if len(image.shape) == 2:
             image = image[:, :, np.newaxis]
 
-        if channels != 'last':
-            image = np.moveaxis(image, -1, 0)
+        if channels:
```

Comment:
True and False are not as descriptive as 'first' and 'last'

Reply:
Imagine we have images with `channels first` format.   
And now we want to convert to `channels last` format for `plt.imshow()` for example.
Currently it seems impossible.   
Instead we can use `channels` argument as the flag for swapping the first and last dimensions.

Reply:
It is excessively implicit. Reconsider all the options possible (e.g. images might be stored as PIL.Images or as arrays, with channels first or last and we want to convert them to channels first or last) and provide explicit parameters (with non-confusing names).